testsuite/fpl_types/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_test(fixed_list_types_test ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/fixed_list_types_test.py)
```

Comment:
Please make sure that the output files (if any) get not written to `${CMAKE_CURRENT_SOURCE_DIR}`, but `${CMAKE_CURRENT_BINARY_DIR}`.